tools/pv_tree_generator/README.MD

```diff
@@ -3,11 +3,15 @@
  **To generate the json file**
  
         sh ./generate_json.sh
+        
 **To run the test** 
 
         sh ./PVtree_test.sh
```

Comment:
file name should be all lower case: pvtree_test.sh

Reply:
got it.